python/src/QuantileConfidence_doc.i.in

```diff
@@ -0,0 +1,440 @@
+
+.. math::
+
+    \argmin_{0 \leq k_1 \leq k_2 \leq n - 1 \, | \, \Pset(k_1 < X \leq k_2) \geq \beta} \Pset(k_1 < X \leq k_2) | 
+
+where :math:`X \sim \cB(n, \alpha)`.
+These ranks exist only if:
+
+.. math::
+
+    1 - \alpha^n - (1-\alpha)^n \ge \beta.
+
+Parameters
+----------
+size : int
+    Sample size :math:`n`
+
+Returns
+-------
+ranks : :class:`~openturns.Indices` of size 2
+    Pair of lower and upper ranks :math:`(k_1, k_2)` such that :math:`0 \leq k_1 \leq k_2 \leq n - 1`.
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> k1, k2 = algo.computeBilateralRank(100)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeUnilateralConfidenceInterval
+"Evaluate an unilateral confidence interval of a quantile.
+
+The lower tail confidence interval is given by the order statistics:
+
+.. math::
+
+    [X_{(k_{low})}, +\infty]
+
+and the upper tail confidence interval is given by the order statistics:
+
+.. math::
+
+    [-\infty, X_{(k_{up})}]
+
+where the lower or upper ranks :math:`k_{low}, k_{up}` are given by :meth:`computeUnilateralRank`
+
+Parameters
+----------
+sample : 2-d sequence of float
+    Quantile level
+tail : bool, optional
+    Whether to return the upper or lower tail confidence interval. Default is False.
+
+Returns
+-------
+ci : :class:`~openturns.Interval`
+    Quantile confidence interval
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> sample = ot.Gumbel().getSample(100)
+>>> ci = algo.computeUnilateralConfidenceInterval(sample)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeBilateralConfidenceInterval
+"Evaluate a bilateral confidence interval of a quantile.
+
+The confidence interval is given by the order statistics:
+
+.. math::
+
+    [X_{(k_1)}, X_{(k_2)}]
+
+where the ranks :math:`k_1, k_2` are given by :meth:`computeBilateralRank`
+
+Parameters
+----------
+sample : 2-d sequence of float
+    Sample of the variable :math:`X`
+
+Returns
+-------
+ci : :class:`~openturns.Interval`
+    Quantile confidence interval
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> sample = ot.Gumbel().getSample(60)
+>>> ci = algo.computeBilateralConfidenceInterval(sample)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeUnilateralMinimumSampleSize
+"Evaluate the minimum size of the sample for the unilateral case.
+
```

Comment:
The purpose of this method is unclear. Please write the property of the integer we search for.